app/common/gristUrls.ts

```diff
@@ -90,6 +90,7 @@ export const commonUrls = {
   helpFilteringReferenceChoices: "https://support.getgrist.com/col-refs/#filtering-reference-choices-in-dropdown",
   helpSandboxing: "https://support.getgrist.com/self-managed/#how-do-i-sandbox-documents",
   helpAPI: 'https://support.getgrist.com/api',
+  helpDocumentType: 'https://support.getgrist.com/creating-doc/',
```

Comment:
It the resource page were I think to add some doc about this new functionality in document settings. Feel free to indicate a better place.